controllers/app.controllers.js

```diff
@@ -62,4 +62,14 @@ exports.createCommentByReviewId = (req, res, next) => {
     .catch((err) => {
         next(err)
     })
+}
+
+exports.removeCommentById = (req, res, next) => {
+    const { comment_id } = req.params;
+    deleteCommentById(comment_id).then((body) => {
+        res.status(204).send(body)
```

Comment:
express won't send anything here so we don't need to invoke the callback with anything or include anything in the send invocation